kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -28,13 +28,29 @@
       </div>
 
       <div class="create">
-        <user-create-modal></user-create-modal>
+        <icon-button @click="openCreateModal" class="add-user-button" text="Add New" :primary="false">
+          <svg class="add-user" src="../icons/add_new_user.svg" role="presentation"></svg>
+        </icon-button>
       </div>
 
     </div>
 
     <hr>
 
+    <!-- Modals -->
+    <template v-if="editingUser">
```

Comment:
It was originally there because I had included comments on top of the modal. I'll get rid of it.